app/scripts/modules/appengine/appengine.module.js

```diff
@@ -0,0 +1,42 @@
+import {module} from 'angular';
+
+import {APPENGINE_CACHE_CONFIGURER} from './cache/cacheConfigurer.service';
+import {APPENGINE_CLONE_SERVER_GROUP_CTRL} from './serverGroup/configure/wizard/cloneServerGroup.controller';
+import {APPENGINE_HELP_CONTENTS_REGISTRY} from './helpContents/appengineHelpContents.ts';
+import {APPENGINE_SERVER_GROUP_BASIC_SETTINGS_CTRL} from './serverGroup/configure/wizard/basicSettings.controller';
+import {APPENGINE_SERVER_GROUP_COMMAND_BUILDER} from './serverGroup/configure/serverGroupCommandBuilder.service';
+import {APPENGINE_SERVER_GROUP_DETAILS_CONTROLLER} from './serverGroup/details/appengine.details.controller';
+import {APPENGINE_SERVER_GROUP_TRANSFORMER} from './serverGroup/transformer';
+
+let templates = require.context('./', true, /\.html$/);
```

Comment:
I think you might need to install the typings for `webpack-env`(`typings install dt~webpack-env --global --save-dev`) - give that a shot and let me know if it fixes this